### ISSUE TITLE
'Non-existent config key: MODEL.MASKIOU_ON' while running demo.py

### ISSUE THREAD

**Issue by Auth0rM0rgan** [youngwanLEE/centermask2#16]
Hey @youngwanLEE, 
I'm trying to run demo.py but I'm getting an error. Can you help me how can I solve it?
```
[03/11 17:23:10 detectron2]: Arguments: Namespace(confidence_threshold=0.5, config_file='../configs/centermask/centermask_lite_V_39_eSE_FPN_ms_4x.yaml', input=None, opts=[], output=None, video_input=None, webcam=True)
WARNING [03/11 17:23:10 d2.config.compat]: Config '../configs/centermask/centermask_lite_V_39_eSE_FPN_ms_4x.yaml' has no VERSION. Assuming it to be compatible with latest v2.
Traceback (most recent call last):
  File "demo.py", line 72, in <module>
    cfg = setup_cfg(args)
  File "demo.py", line 23, in setup_cfg
    cfg.merge_from_file(args.config_file)
  File "/home/media4us/PycharmProjects/detectron2/detectron2/config/config.py", line 47, in merge_from_file
    self.merge_from_other_cfg(loaded_cfg)
  File "/home/media4us/anaconda3/lib/python3.7/site-packages/fvcore/common/config.py", line 121, in merge_from_other_cfg
    return super().merge_from_other_cfg(cfg_other)
  File "/home/media4us/anaconda3/lib/python3.7/site-packages/yacs/config.py", line 217, in merge_from_other_cfg
    _merge_a_into_b(cfg_other, self, self, [])
  File "/home/media4us/anaconda3/lib/python3.7/site-packages/yacs/config.py", line 460, in _merge_a_into_b
    _merge_a_into_b(v, b[k], root, key_list + [k])
  File "/home/media4us/anaconda3/lib/python3.7/site-packages/yacs/config.py", line 473, in _merge_a_into_b
    raise KeyError("Non-existent config key: {}".format(full_key))
KeyError: 'Non-existent config key: MODEL.MASKIOU_ON' 
```

I have cloned and installed detectron2 and adet successfully without getting any errors. 

This is the command that I've used to run demo.py

`python demo.py --webcam  --config-file ../configs/centermask/centermask_lite_V_39_eSE_FPN_ms_4x.yaml
`

Thanks!

**Comment by denverdash** [comment]
Having same problem here.

**Comment by Rm1n90** [comment]
@youngwanLEE, I think your demo.py is broken. Having the same problem here.

**Comment by youngwanLEE** [comment]
@Auth0rM0rgan @denverdash @Rm1n90 
I fixed [demo.py](https://github.com/youngwanLEE/centermask2/blob/master/demo/demo.py). 
Would you like to try again?

**Comment by Auth0rM0rgan** [comment]
Hey @youngwanLEE, Still demo.py not working

my command line: 

`python demo.py --config-file ../configs/centermask/centermask_V_39_eSE_FPN_ms_3x.yaml --video-input /home/xx/Downloads/avg.mp4`

The error I'm getting is:

`  File "demo.py", line 14, in <module>
    from centermask.config import get_cfg
ModuleNotFoundError: No module named 'centermask'
`
Thanks!

**Comment by Rm1n90** [comment]
@youngwanLEE having same error as @Auth0rM0rgan!

**Comment by youngwanLEE** [comment]
@Rm1n90 @Auth0rM0rgan 

Would you like to try again?

**Comment by Auth0rM0rgan** [comment]
@youngwanLEE,
Thanks, demo.py is working now.
Also, I have a question regard to Fps on 1080p video. Which one of your lite pretrained model is suitable to reach 30fps for instance segmentation on 1080P video? 
I have tried all the lite pretrained models on my 1080p video but none of them could reach to 30fps. I have also tried the YOLACT 550 with ResNet50 and I was able to achieve almost ~25fps on 1080p video. according to your paper, CenterMask is faster than YOLACT but in my test yolact do the instance segmentation faster. any tips? 

Thanks!

**Comment by youngwanLEE** [comment]
@Auth0rM0rgan


While Our demo program originates from detectron2, YOLACT visualization tools are optimized well for better speed. 

Inference time in many papers is usually measured using the only model forward time not including visualization step.

We also follow detectron2 rule.

I recommend you use [lite models](https://github.com/youngwanLEE/centermask2#lightweight-vovnet-backbone) such as [V-19-slim](https://github.com/youngwanLEE/centermask2/blob/master/configs/centermask/centermask_lite_V_19_slim_eSE_FPN_ms_4x.yaml) or [V-19-slim-dw](https://github.com/youngwanLEE/centermask2/blob/master/configs/centermask/centermask_lite_V_19_slim_dw_eSE_FPN_ms_4x.yaml).

And,

if you lower the '[MAX_SIZE_TEST](https://github.com/youngwanLEE/centermask2/blob/master/configs/centermask/Base-CenterMask-Lite-VoVNet.yaml#L43)', you can get faster speed.

> python demo/demo.py  ... --opts INPUT.MAX_SIZE_TEST 512

In conclusion,

1. Select well-balanced (speed and accuracy) models
2. Lower `MAX_SIZE_TEST`
3. Search the best combination between the model and `MAX_SIZE_TEST` to the extent you accept.

**Comment by Auth0rM0rgan** [comment]
Hey @youngwanLEE, Thanks for the tips but when I set the `INPUT.MAX_SIZE_TEST` i'm not getting any output (I mean code is working but masks are not overlay on the objects)

`python demo.py --video-input /home/Downloads/tennis_720.mp4 --config-file ../configs/centermask/centermask_V_39_eSE_FPN_ms_3x.yaml --opts MODEL.WEIGHTS ../centermask2-V-39-eSE-FPN-ms-3x.pth INPUT.MAX_SIZE_TEST 128
`

When my command line is without `INPUT.MAX_SIZE_TEST` everything works fine and masks are overlaying on top of objects. Thanks!

**Comment by youngwanLEE** [comment]
@Auth0rM0rgan 

As I said, you should fine the best `MAX_SIZE_TEST` suitable for your task (e.g., speed, resolution, etc).

I think `MAX_SIZE_TEST` of 128 is too small to detect.

How about 512?
